src/CommunityToolkit.Maui.UnitTests/Mocks/AnimationExtensions.cs

```diff
@@ -54,7 +54,7 @@ public object GetService(Type serviceType)
 			}
 		}
 
-		class AsyncTicker : Ticker
+		class AsyncTicker : Ticker, IDisposable
```

Comment:
Let's just make this a `sealed class` and then we don't need to implement this inheritable IDisposable pattern.

Reply:
https://stackoverflow.com/a/1418998/5953643